[2024-03-21T19:05:08.965+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ETL_Flow.insert_data_from_files manual__2024-03-16T06:27:40.650704+00:00 [queued]>
[2024-03-21T19:05:08.993+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ETL_Flow.insert_data_from_files manual__2024-03-16T06:27:40.650704+00:00 [queued]>
[2024-03-21T19:05:08.995+0000] {taskinstance.py:2193} INFO - Starting attempt 12 of 17
[2024-03-21T19:05:09.037+0000] {taskinstance.py:2214} INFO - Executing <Task(PythonOperator): insert_data_from_files> on 2024-03-16 06:27:40.650704+00:00
[2024-03-21T19:05:09.049+0000] {standard_task_runner.py:60} INFO - Started process 559 to run task
[2024-03-21T19:05:09.058+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'ETL_Flow', 'insert_data_from_files', 'manual__2024-03-16T06:27:40.650704+00:00', '--job-id', '2139', '--raw', '--subdir', 'DAGS_FOLDER/etl2.py', '--cfg-path', '/tmp/tmpwii3e_lv']
[2024-03-21T19:05:09.066+0000] {standard_task_runner.py:88} INFO - Job 2139: Subtask insert_data_from_files
[2024-03-21T19:05:09.228+0000] {task_command.py:423} INFO - Running <TaskInstance: ETL_Flow.insert_data_from_files manual__2024-03-16T06:27:40.650704+00:00 [running]> on host 87d0282d7958
[2024-03-21T19:05:10.519+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='ETL_Flow' AIRFLOW_CTX_TASK_ID='insert_data_from_files' AIRFLOW_CTX_EXECUTION_DATE='2024-03-16T06:27:40.650704+00:00' AIRFLOW_CTX_TRY_NUMBER='12' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-16T06:27:40.650704+00:00'
[2024-03-21T19:05:12.455+0000] {base.py:83} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-03-21T19:05:13.666+0000] {sql.py:572} INFO - Done loading. Loaded a total of 92 rows into MRR_Dim_Customers
[2024-03-21T19:05:13.822+0000] {base.py:83} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-03-21T19:05:14.139+0000] {sql.py:572} INFO - Done loading. Loaded a total of 47 rows into MRR_Dim_Employees
[2024-03-21T19:05:14.276+0000] {base.py:83} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-03-21T19:05:14.780+0000] {sql.py:572} INFO - Done loading. Loaded a total of 77 rows into MRR_Dim_Products
[2024-03-21T19:05:16.074+0000] {base.py:83} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-03-21T19:05:25.393+0000] {sql.py:572} INFO - Done loading. Loaded a total of 836 rows into MRR_Fact_Orders
[2024-03-21T19:05:25.922+0000] {base.py:83} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-03-21T19:05:36.402+0000] {sql.py:568} INFO - Loaded 1000 rows into MRR_Fact_Details so far
[2024-03-21T19:05:41.949+0000] {sql.py:568} INFO - Loaded 2000 rows into MRR_Fact_Details so far
[2024-03-21T19:05:42.499+0000] {sql.py:572} INFO - Done loading. Loaded a total of 2172 rows into MRR_Fact_Details
[2024-03-21T19:05:42.502+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-03-21T19:05:42.529+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=ETL_Flow, task_id=insert_data_from_files, execution_date=20240316T062740, start_date=20240321T190508, end_date=20240321T190542
[2024-03-21T19:05:42.622+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-21T19:05:42.668+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/models/baseoperator.py:1201: AirflowProviderDeprecationWarning: Call to deprecated class PostgresOperator. (Please use `***.providers.common.sql.operators.sql.SQLExecuteQueryOperator`.Also, you can provide `hook_params={'schema': <database>}`.)
  result = cls.__new__(cls)

[2024-03-21T19:05:42.798+0000] {taskinstance.py:3309} INFO - 1 downstream tasks scheduled from follow-on schedule check
